package.json

```diff
@@ -116,5 +116,8 @@
     "webpack": "^4.44.2",
     "webpack-cli": "^3.3.12",
     "webpack-distsize": "^0.1.0"
+  },
+  "dependencies": {
+    "date-fns": "^2.19.0"
```

Comment:
All dependencies should either be peerDependencies (so it can be de-duped by the parent app) or devDependencies (so it can be bundled in).

Can you try making it a devDependency to see the size impact? If it's big, we can make it a peer dep.